packages/ui/test/setup.ts

```diff
@@ -1 +1,34 @@
 import '@testing-library/jest-dom'
+import BN from 'bn.js'
+
+declare global {
+  // eslint-disable-next-line @typescript-eslint/no-namespace
```

Comment:
I'd consider this a placeholder until we see if we don't wan to create a library for BN matchers :thinking: